includes/wikia/services/MediaQueryService.class.php

```diff
@@ -427,12 +432,13 @@ public function getVideoList( $type = 'all', $limit = 0, $page = 1, $providers =
 				break;
 
 			case self::SORT_TRENDING_FIRST:
+			case self::SORT_TRENDING_FIRST_LEGACY:
 				$query->ORDER_BY( self::DB_TRENDING_COLUMN )->DESC();
 				break;
 
 			default:
 				throw new InvalidArgumentException( "\$sort was none of '" . self::SORT_RECENT_FIRST . "', '"
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the code after this "throw". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1763)